src/uniparc/components/entry/Entry.tsx

```diff
@@ -52,7 +52,7 @@ const Entry: FC = () => {
     redirectedTo,
     headers,
   } = useDataApi<UniParcAPIModel>(baseURL);
-  // const facetData = useDataApi<Facet[]>(`${baseURL}/facets`);
+  // const facetData = useDataApi<FacetObject[]>(`${baseURL}/facets`);
```

Comment:
Are these two comments (also L34) needed? If so, could you comment with a `TODO:` to flag for future plans.

Reply:
This can actually go, I assumed I was asking backend for these facets in UniParc, but they need to be generated on the client